package.json

```diff
@@ -31,6 +31,7 @@
     "@angular/platform-browser": "~7.0.0",
     "@angular/router": "~7.0.0",
     "@swimlane/ngx-datatable": "^14.0.0",
+    "@swimlane/ngx-ui": "^25.1.0",
```

Comment:
Is this needed?

Reply:
I followed the installation instruction on its README:
`npm i @swimlane/ngx-ui --S` 

I can remove this though